client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -115,15 +116,15 @@ export const useCommandsArrayWpcom = ( {
 
 	const siteFilters = {
 		hostingEnabled: {
-			capabilityFilter: 'manage_options',
+			capabilityFilter: SiteCapabilities.MANAGE_OPTIONS,
```

Comment:
What if we defined type as `keyof typeof SiteDetailsCapabilities` and skipped the new enum to avoid defining capabilities in two places?

Reply:
That's a good suggestion! I'll do that!

Reply:
Initially, I thought that I could use your suggestion to somehow create easily an object out of it.
I don't mind about the type so much, as types are not available at runtime. I wanted to define it like that to avoid magic strings scattered all over the place, as we have them now, and to have a centralized object for capabilities that can be reused.
So I will not implement it like that, except if you have something in mind that I can use to achieve what I described easily, to avoid repeating the capabilities defined in the `SiteDetailsCapabilities` interface.

Reply:
That sounds good. Thanks for providing more details.